plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -7616,6 +7617,59 @@ public void testSelectFromPrestoViewReferencingHiveTableWithTimestamps()
         assertThat(query(nanosSessions, "SELECT ts FROM hive_timestamp_nanos.tpch." + prestoViewNameNanos)).matches("VALUES TIMESTAMP '1990-01-02 12:13:14.123000000'");
     }
 
+    @Test(dataProvider = "legalUseColumnNamesProvider")
+    public void testUseColumnNames(HiveStorageFormat format, boolean formatUseColumnNames)
+    {
+        String lcFormat = format.name().toLowerCase(Locale.ROOT);
+        Session.SessionBuilder builder = Session.builder(getQueryRunner().getDefaultSession())
+                .setIdentity(Identity.forUser("hive")
```

Comment:
I don't think this `setIdentity()` call is needed. (it seems redundant in all the other tests)

Reply:
Removed.